libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.h

```diff
@@ -247,6 +255,13 @@ FOUNDATION_EXTERN NSString * const kSFUserInfoAddlOptionsKey;
 
 @end
 
+/** User Information for post logout notifications.
+ */
+@interface SFNotificationUserInfo : NSObject
```

Comment:
Should we consider instead just using `SFUserAccountIdentity`?  CommunityID doesn't explicitly come into play for logout, and SFUserAccountIdentity otherwise actually has all of the information identifying a user.

Reply:
@khawkins Good idea. I can weave SFUserAccountIdentity into the core of this change, but the approach I took was based on the assumption that, more than just these id's maybe required in the future. Nevertheless I could add SFUserAccountIdentity to the SFNotificationUserInfo object. what do you think?

Reply:
The future-proofing idea is a good one.  Yeah, maybe let's make the account identity a member of that object?